### PR TITLE
FAT-1517 Karate Tests: Create MARC authority records via data import

### PR DESCRIPTION
**Pull request by YahorSviarchkou**
### Learning

[FAT-1517 Create MARC authority records via data import](https://issues.folio.org/browse/FAT-1517)

### CODE REVIEW

**Review by ALiemeshka — APPROVED** [review]

File: data-import/src/main/resources/folijet/data-import/features/create-records.feature [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+    Given call read(utilFeature+'@ImportRecord') { fileName:'marcAuthorityInvalid1XX', jobName:'createAuthority' }
+    Then match status != 'ERROR'
+
+    Given path '/source-storage/source-records'
+    And param recordType = 'MARC_AUTHORITY'
+    And param snapshotId = jobExecutionId
+    And headers headersUser
+    And retry until response.totalRecords > 0 && karate.sizeOf(response.sourceRecords[0].externalIdsHolder) > 0
+    When method get
+    Then status 200
```
Comment (psmagin) [inline data-import/src/main/resources/folijet/data-import/features/create-records.feature:41]: If data-import is failed then why do you check source-storage contains some record. You should check that record doesn't exist

File: data-import/src/main/resources/folijet/data-import/features/create-records.feature [hunk 2]
```diff
@@ -0,0 +1,50 @@
+Feature: Create records
```
Comment (psmagin) [inline data-import/src/main/resources/folijet/data-import/features/create-records.feature:1]:
```suggestion
Feature: Create marc authority records
```

Comment (psmagin) [reply data-import/src/main/resources/folijet/data-import/features/create-records.feature:1]: and the save for file name

**Review by psmagin — CHANGES_REQUESTED** [review]

File: data-import/src/main/resources/folijet/data-import/features/create-records.feature (hunk 1 above)
Comment (YahorSviarchkou) [reply data-import/src/main/resources/folijet/data-import/features/create-records.feature:41]: > If data-import is failed then why do you check source-storage contains some record. You should check that record doesn't exist

Date import can create authority record, even if the record contains not valid 1xx

**Review by psmagin — APPROVED** [review]

**Pull request merged**